SwrveSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java

```diff
@@ -16,6 +18,20 @@
  * Configuration for the Swrve SDK.
  */
 public abstract class SwrveConfigBase {
+
+    /**
+     * Default Log Tag
+     */
+
+    private final String LOG_TAG = "SwrveConfigBase";
+
+    /**
+     * List of supported stacks
+     */
+    public enum SwrveStack{
+        EU, US, ACTIVISION, FEATURESTACK
```

Comment:
This is something I asked about and didn't get an answer.

What is the security concern in exposing featurestacks via api? Is there something damaging developers can do?

Also, if it isn't ok to use featurestacks, why is it ok to expose activision?

@robertclancy Can I grab your opinion on this?